### DISCUSSION TITLE
Asset search goes over all assets

### DISCUSSION THREAD

**Discussion by quinadmin** [snipe/snipe-it#13460] [Q&A] [answered]
Hi,

I am not sure if its a bug or a error I made.

When I select "all ready to deploy" Assets, an then do a search, it searches over all assets and not only about "ready to deploy" ones.

I cannot reproduce this on the demo on https://demo.snipeitapp.com/ so somehow it seems to be a problem in my installation.

I have added screenshots to a issue someone else made but its exactly the same problem as mine i think.
 
https://github.com/snipe/snipe-it/issues/13103#issuecomment-1674691623

Does anybody had the same problem or has an idea how to fix this?

**Comment by snipe** [comment, marked as answer]
I believe this was fixed in #13227 - if you upgrade, this issue should go away.

**Reply by quinadmin** [reply]
Yes! Not Problem with 6.1.2! Thank you!
